lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/hmrcKbvCri/ContractTest.java

```diff
@@ -0,0 +1,675 @@
+                    },
+                    {
+                        "kbvResponseMode": "multiple_choice",
+                        "kbvQuality": 3,
+                        "checkMethod": "kbv"
+                    },
+                    {
+                        "kbvResponseMode": "multiple_choice",
+                        "kbvQuality": 2,
+                        "checkMethod": "kbv"
+                    }
+                    ],
+                    "verificationScore": 2,
+                    "txn": "dummyTxn",
+                    "type": "IdentityCheck"
+                }]
+              }
+            }
+            """;
+
+    private static final String FAILED_VC_BODY =
+            """
+            {
+              "iss": "dummyHmrcKbvComponentId",
+              "sub": "test-subject",
+              "nbf": 4070908800,
+              "exp": 4070909400,
+              "vc": {
+                "@context": [
+                  "https://www.w3.org/2018/credentials/v1",
+                  "https://vocab.london.cloudapps.digital/contexts/identity-v1.jsonld"
+                ],
+                "type": [
+                  "VerifiableCredential",
+                  "IdentityCheckCredential"
+                ],
+                "credentialSubject": {
+                  "socialSecurityRecord": [
+                    {
+                       "personalNumber": "AA000003D"
+                    }
+                  ],
+                  "name": [
+                    {
+                      "nameParts": [
+                        {
+                          "type": "GivenName",
+                          "value": "Mary"
+                        },
+                        {
+                          "type": "FamilyName",
+                          "value": "Watson"
+                        }
+                      ]
+                    }
+                  ],
+                  "birthDate": [
+                    {
+                      "value": "1932-02-25"
+                    }
+                  ]
+                },
+                "evidence": [{
+                    "checkDetails": [{
+                        "kbvResponseMode": "free_text",
+                        "kbvQuality": 3,
+                        "checkMethod": "kbv"
+                    }],
+                    "failedCheckDetails": [{
+                        "kbvResponseMode": "free_text",
+                        "kbvQuality": 3,
+                        "checkMethod": "kbv"
+                    },
+                    {
+                        "kbvResponseMode": "multiple_choice",
+                        "kbvQuality": 2,
+                        "checkMethod": "kbv"
+                    }],
+                    "ci": ["V03"],
+                    "verificationScore": 0,
+                    "txn": "dummyTxn",
+                    "type": "IdentityCheck"
+                }]
+              }
+            }
+            """;
```

Comment:
A thought for a possible extra test case. IIRC the user has to get 3 out of 4 questions right. If they do get asked 4 questions do we get 4 in the evidence section or is the wrong answer ignored?